### ISSUE TITLE
concluding license does not update buckets

### ISSUE THREAD

**Issue by yaobinshi** [fossology/fossology#354]
---

Author Name: **Bob Gobeille** 
Original Redmine Issue: 7875, http://www.fossology.org/issues/7875
Original Date: 2014/10/17
Original Assignee: Bob Gobeille

---

In 2.5 when you updated a license, a new bucket scan would happen on that file.  This seems to have been lost with the new concluded license code.

**Comment by mcjaeger** [comment]
restest with current master required

**Comment by mcjaeger** [comment]
fair enough, because since the bucket agent is not scheduled after a license was concluded for a file, the buckets cannot be updated (admitting that I should have thought about the issue before asking for retest).

Scheduling the bucket agent after each license conclusion does not appear feasible for me. Rather a feature that tells the user in the bucket browser that the view is outdated due to new results. Would that work.

Because this sounds more like a feature than a bug I am rescheduling it for 3.1.

**Comment by shaheemazmalmmd** [comment]
Bucket agent will have new representation based on license category/type which is underdevelopment this GSoC.  SEE : https://github.com/fossology/fossology/issues/3630 

closing this issue. Please reopen a new issue if anything in addition needed apart from features mentioned in PR#3630
